### DISCUSSION TITLE
Is there any other way to get the prod key and title key without a switch?

### DISCUSSION THREAD

**Discussion by Raaf011** [skyline-emu/skyline#1996] [Q&A] [answered]
I really want to play but i dont have switch

**Comment by PixelyIon** [comment, marked as answer]
There isn't a way to do so legally; Nintendo only distributes keys via the Switch, so you can only dump them from it.

**Comment by Tanin13** [comment]
I have a way but uh i can't say it here tho lol

**Reply by KhairulNnN** [reply]
Do you mind i contact you via DM ?
